apps/console/src/extensions/i18n/models/extensions.ts

```diff
@@ -243,6 +243,8 @@ export interface Extensions {
                 groupAttributeLabel: string;
                 groupAttributeHint: string;
                 groupAttributePlaceholder: string;
+                groupAttributeMessage1: string;
+                groupAttributeMessage2: string;
```

Comment:
better to have a different name for this variables

Reply:
Will be addressed through a followup PR.